frontend/src/components/nodes/NodesMap.tsx

```diff
@@ -174,36 +126,44 @@ class NodesMap extends React.Component<State> {
     // Prettier ruined the entire indentation that i made for this
     return `<div className="hoverinfo" style="border: none; text-align: left; padding: 20px 0px 0px; box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.16); border-radius: 8px; color: white; background-color: #343A40; max-width: 300px">
         <div style="color: #8DD4BD; font-size: 14px; line-height: 14px; letter-spacing: 0.4px; font-weight: bold; font-family: BwSeidoRound; padding:0 20px 8px"> @${
-          data.nodeInfo.accountId
+          data.nodeInfo.account_id || data.nodeInfo.accountId
```

Comment:
Why did we introduce this inconsistency? Can we avoid it?

Reply:
^^^